internal/schema2/numa.go

```diff
@@ -0,0 +1,16 @@
+/*
+ * HCS API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: 2.4
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package hcsschema
+
+type Numa struct {
+	VirtualNodeCount       int32         `json:"VirtualNodeCount,omitempty"`
```

Comment:
Swagger for some reason doesn't always output the correct type for fields on a schema. From a conversation with Stefan W. once upon a time:

```
also, the uint32 vs int32 might be a limitation of our code generation... I think uint32 is not a type in json schema
so our uint32 get turned into "int" in json schema, which then comes out as int32 in go
```

The field is actually a uint8 in the HCS mars file. I think you should update accordingly. Here and to the rest of the PR.

Reply:
You will need to check the fields of the other schemas you've added as well.

Reply:
Correct I've noticed. Didn't know if we wanted to leave this behavior as that's what we've done for all the other files but it's much nicer if these are unsigned ints.